src/_angular/font-setting/dtv-font-setting.js

```diff
@@ -383,6 +410,51 @@
               sheet.addRule("@font-face", rule);
             }
           }
+
+          function addVerticalAlignButtons(editor) {
+            editor.addButton("aligntop", {
+              image: "//s3.amazonaws.com/Rise-Images/Icons/align-top.svg",
```

Comment:
@stulees Can this be part of the repo and added to the `dist` folder as part of the build?